app/controllers/concerns/controller_concern.rb

```diff
@@ -9,4 +9,12 @@ def component_name_from_params(params)
     key = component_key(params)
     key.gsub('_id', '').split('_').map(&:titleize).join
   end
+
+  def certificate_issuer_common_name(certificate)
```

Comment:
I think these methods belong to the CertificateConcern
**certificate_issuer_find** 
**certificate_issuer_common_name**
Or at the very least they seem out of place here. If though the class is call ControllerConcern it seemed to be doing stuff explicitly on the component.